### PR TITLE
CAMS-401 remove USTP urls from mock issuer

### PR DESCRIPTION
**Pull request by amorrow-flexion**
Jira ticket: CAMS-401

# Purpose

We have Okta for USTP now, so we do not need mockIssuer URLS for those 2 environments any more

# Major Changes

removal of USTP Urls

# Testing/Validation

N/A

### CODE REVIEW

File: .github/workflows/continuous-deployment.yml [hunk 1]
```diff
@@ -178,6 +178,7 @@ jobs:
       environmentHash: ${{ needs.setup.outputs.environmentHash }}
       cosmosDbClientId: ${{ needs.deploy.outputs.cosmosDbClientId }}
       cosmosDbPrincipalId: ${{ needs.deploy.outputs.cosmosDbPrincipalId }}
+      mockAuth: ${{ vars.MOCK_AUTH }}
```
Comment (amorrow-flexion) [inline .github/workflows/continuous-deployment.yml:181]: I don't see a GHA var setup for this. What is the intent of it?

File: backend/functions/lib/configs/authorization-configuration.ts [hunk 2]
```diff
@@ -3,13 +3,16 @@ import { AuthorizationConfig } from '../adapters/types/authorization';
 
 dotenv.config();
 
+const doMockAuth = process.env.MOCK_AUTH === 'true';
```
Comment (amorrow-flexion) [inline backend/functions/lib/configs/authorization-configuration.ts:6]: Why bother setting up another env when we can leverage provider and do

`const doMockAuth = process.env.CAMS_LOGIN_PROVIDER === 'mock';`

File: .github/workflows/sub-deploy-code-slot.yml [hunk 3]
```diff
@@ -112,7 +115,7 @@ jobs:
           -n ${{ inputs.apiName }} \
           --src ./${{ inputs.apiName }}.zip \
           --slotName ${{ inputs.slotName }} \
-          --settings "INFO_SHA=${{ github.sha }}"
+          --settings "INFO_SHA=${{ github.sha }} MOCK_AUTH=${{ inputs.mockAuth }}"
```
Comment (amorrow-flexion) [inline .github/workflows/sub-deploy-code-slot.yml:118]: I'm not sure we need this here. We read from React env on the frontend, I don't think we would ever access this setting

File: backend/functions/lib/configs/authorization-configuration.ts (hunk 2 above)
Comment (btposey) [reply backend/functions/lib/configs/authorization-configuration.ts:6]: We could have done it that way too. There was code already in place to look for the MOCK_AUTH env var so we built off that.

**Review by btposey — APPROVED** [review]

**Review by btposey — APPROVED** [review]

**Pull request merged**